UITests/BookmarksPanelTests.swift

```diff
@@ -61,20 +61,40 @@ class BookmarksPanelTests: KIFTestCase {
         EarlGrey.select(elementWithMatcher: grey_accessibilityLabel("Desktop Bookmarks")).perform(grey_tap())
         EarlGrey.select(elementWithMatcher: grey_accessibilityLabel("Bookmarks Toolbar")).perform(grey_tap())
         EarlGrey.select(elementWithMatcher: grey_accessibilityLabel("AAA"))
-        .assert(grey_sufficientlyVisible())
+            .assert(grey_sufficientlyVisible())
         EarlGrey.select(elementWithMatcher: grey_accessibilityLabel("Some Livemark"))
             .assert(grey_sufficientlyVisible())
         
         // When we tap the livemark, we load the siteURI.
         EarlGrey.select(elementWithMatcher: grey_accessibilityLabel("Some Livemark")).perform(grey_tap())
         
         // … so we show the truncated URL.
-        // Strangely, earlgrey cannot find the label in buddybuild, but passes locally. 
+        // Strangely, earlgrey cannot find the label in buddybuild, but passes locally.
         // Using KIF for this check for now.
         tester().waitForView(withAccessibilityLabel: "https://people-mozilla.org/~npark")
         
     }
 
+    private func navigateBackInTableView() {
+        EarlGrey.select(elementWithMatcher: grey_accessibilityLabel("Bookmarks")).inRoot(grey_kindOfClass(NSClassFromString("UITableView")!)).perform(grey_tap())
+    }
+    
+    private func navigateFolder(withTitle title: String) {
+        EarlGrey.select(elementWithMatcher: grey_accessibilityLabel(title)).perform(grey_tap())
+    }
+    
+    private func makeBookmark(guid: String, parentID: String, title: String) -> BookmarkMirrorItem {
```

Comment:
We have a little type alias hanging around: `GUID`. Use that instead of `String` where appropriate, like here.

Reply:
thanks will do!

Reply:
Functions like this are changed to the much clearer: 
`func makeFolder(guid: GUID, parentID: GUID, title: String, children: [GUID]) -> BookmarkMirrorItem {`